core/src/main/scala/kafka/admin/ConsumerGroupCommand.scala

```diff
@@ -87,17 +87,21 @@ object ConsumerGroupCommand extends Logging {
       }
     } catch {
       case e: Throwable =>
-        printError(s"Executing consumer group command failed due to ${e.getMessage}", Some(e))
+        printError(s"Executing consumer group command failed (${e.getMessage}) ${System.lineSeparator()}" +
+          s"${if (e.getCause != null) e.getCause}", Some(e))
     } finally {
       consumerGroupService.close()
     }
   }
 
   val MISSING_COLUMN_VALUE = "-"
 
-  def printError(msg: String, e: Option[Throwable] = None): Unit = {
+  def printError(msg: String, t: Option[Throwable] = None): Unit = {
     println(s"Error: $msg")
-    e.foreach(debug("Exception in consumer group command", _))
+    t.foreach(e => {
+      debug("Exception in consumer group command: ", e)
+      debug(Utils.stackTrace(e))
```

Comment:
Why do we need this? Won't the line above print the stack trace already? Maybe the intent is to use `println` instead of `debug`?

On a side note, maybe `printError` should be using stderr?

Reply:
The line above does not print a stack trace. Example output:
```
Exception in consumer group command: org.apache.kafka.common.KafkaException: Failed to construct kafka consumer
````

With the change, we get the stack trace as well:
```
org.apache.kafka.common.KafkaException: Failed to construct kafka consumer
	at org.apache.kafka.clients.consumer.KafkaConsumer.<init>(KafkaConsumer.java:794)
	at org.apache.kafka.clients.consumer.KafkaConsumer.<init>(KafkaConsumer.java:644)
	at org.apache.kafka.clients.consumer.KafkaConsumer.<init>(KafkaConsumer.java:624)
	at kafka.admin.ConsumerGroupCommand$KafkaConsumerGroupService.createNewConsumer(ConsumerGroupCommand.scala:672)
	at kafka.admin.ConsumerGroupCommand$KafkaConsumerGroupService.kafka$admin$ConsumerGroupCommand$KafkaConsumerGroupService$$getConsumer(ConsumerGroupCommand.scala:655)
	at kafka.admin.ConsumerGroupCommand$KafkaConsumerGroupService.getLogStartOffsets(ConsumerGroupCommand.scala:621)
	at kafka.admin.ConsumerGroupCommand$KafkaConsumerGroupService.prepareOffsetsToReset(ConsumerGroupCommand.scala:735)
	at kafka.admin.ConsumerGroupCommand$KafkaConsumerGroupService.resetOffsets(ConsumerGroupCommand.scala:681)
	at kafka.admin.ConsumerGroupCommand$.main(ConsumerGroupCommand.scala:81)
	at kafka.admin.ConsumerGroupCommand.main(ConsumerGroupCommand.scala)
Caused by: org.apache.kafka.common.config.ConfigException: request.timeout.ms should be greater than session.timeout.ms and fetch.max.wait.ms
	at org.apache.kafka.clients.consumer.KafkaConsumer.<init>(KafkaConsumer.java:668)
	... 9 more
 (kafka.admin.ConsumerGroupCommand$)
```

I can change `printError` to use stderr.